operator/v1/types_ingress.go

```diff
@@ -1037,6 +1039,18 @@ type SyslogLoggingDestinationParameters struct {
 // ContainerLoggingDestinationParameters describes parameters for the Container
 // logging destination type.
 type ContainerLoggingDestinationParameters struct {
+	// maxLength is the maximum length of the log message.
+	//
+	// If this field is empty, the maxLength is set to "1024".
+	//
+	// Allowed values are between "480" and "8192".
```

Comment:
Sorry to ask you to change this again, but while we wait for the API reviewers, I can suggest some changes that I anticipate the API reviewers will want:
```suggestion
	// maxLength is the maximum length of log messages.
	// Valid values are integers in the range 480 to 8192, inclusive.
	// When omitted, the default value is 1024.
```
Can you update the godoc for both `ContainerLoggingDestinationParameters` and `SyslogLoggingDestinationParameters` with these changes?